dashboard/app/controllers/sms_controller.rb

```diff
@@ -1,33 +1,43 @@
 require 'twilio-ruby'
 class SmsController < ApplicationController
-  protect_from_forgery except: [:send_to_phone] # the page that posts here is cached
+  protect_from_forgery except: [:send_to_phone, :send_download_url_to_phone] # the page that posts here is cached
 
   # set up a client to talk to the Twilio REST API
   def send_to_phone
     if params[:level_source] && !params[:level_source].empty? && params[:phone] && (level_source = LevelSource.find(params[:level_source]))
-      send_sms(level_source_url(level_source), params[:phone])
+      send_sms_link(level_source_url(level_source), params[:phone])
     elsif params[:channel_id] && params[:phone] && ProjectsController::STANDALONE_PROJECTS.include?(params[:type])
       url =
         if params[:type] == 'weblab'
           "https://codeprojects.org/#{params[:channel_id]}"
         else
           polymorphic_url(["#{params[:type]}_project_share", 'projects'], channel_id: params[:channel_id])
         end
-      send_sms(url, params[:phone])
+      send_sms_link(url, params[:phone])
     else
       head :not_acceptable
     end
   end
 
+  def send_download_url_to_phone
+    body = "Download this app created in Code Studio: #{params[:url]} (reply STOP to stop receiving this)"
```

Comment:
"Install this app created in Code Studio on your Android device" https://expo.io/artifacts/cfcb5471-1ef7-465e-a502-bd73c13c1174 (reply stop to stop receiving this)